pallets/ddc-validator/src/lib.rs

```diff
@@ -911,25 +915,47 @@ pub mod pallet {
 							return Err("signing key not set")
 						}
 						// ToDo: replace local call by a call from `ddc-staking` pallet
-						let _tx_res: Option<(frame_system::offchain::Account<T>, Result<(), ()>)> =
+						let tx_res: Option<(frame_system::offchain::Account<T>, Result<(), ()>)> =
 							signer.send_signed_transaction(|_account| {
 								Call::charge_payments_content_owners {
 									paying_accounts: final_payments.clone(),
 								}
 							});
 
+						for (acc, res) in &tx_res {
```

Comment:
Is `tx_res` an iterator?

Reply:
`std::option::Option<(frame_system::offchain::Account<T>, Result<(), ()>)>`
It has this type, but an iterator is present

Reply:
updated

Reply:
I'm not sure if updated code will not panic. let's sync on this

Reply:
in general you have an iterator over an option, which yield one value if it is some, otherwise none